lib/views/staging-view.js

```diff
@@ -586,6 +606,35 @@ export default class StagingView extends React.Component {
     menu.popup(remote.getCurrentWindow());
   }
 
+  showActionsMenu(event) {
+    event.preventDefault();
+
+    const menu = new Menu();
+
+    const selectedItemCount = this.getSelectedItemFilePaths().length;
```

Comment:
This could also be `this.state.selection.getSelectedItems().size` to save an Array construction, yeah?

(Micro-optimization, not important 😆 )

Reply:
Indeed. I'm all for optimization 👍